tmou-mapa-backend/src/database/db.rs

```diff
@@ -24,3 +24,5 @@ pub trait MessagesDb {
     fn get_messages(&self, team_id: i32, limit: Option<i64>) -> Option<Vec<Message>>;
     fn put_message(&self, message: WebMessage, teams_ids: Vec<i32>) -> TmouResult<()>;
 }
+
+pub trait ItemEditingDb {}
```

Comment:
Pojmenoval bych to jinak. Ale nevím jak. Jen vím, že jinak.

Reply:
Měl jsem AdminDb, taky se mi nelíbilo. Ještě by mohlo být:

* GameEditorDb/GameEditingDb
* DeveloperDb
* CreatorDb

?